package.json

```diff
@@ -9,6 +9,7 @@
   "engines": {
     "node": "^16 || ^18 || ^20"
   },
+  "packageManager": "yarn@3.8.1",
```

Comment:
Turbo will now error if this isn't specified - https://turbo.build/repo/docs/reference/configuration#dangerouslydisablepackagemanagercheck